components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/cmd/BuildCmd.java

```diff
@@ -327,4 +334,22 @@ private void initTarget(String projectName) throws IOException {
         CmdUtils.createDirectory(projectModuleDir, true);
     }
 
+    /**
+     * Invoke call home.
+     *
+     */
+    private void invokeCallhome() {
+        String productHome = CmdUtils.getCLIHome();
+        String trustStoreLocation = productHome + File.separator + RESTServiceConstants.DEFAULT_TRUSTSTORE_PATH;
+        String trustStorePassword = RESTServiceConstants.DEFAULT_TRUSTSTORE_PASS;
+
+        CallHomeInfo callhomeinfo = Util.createCallHomeInfo(productHome, trustStoreLocation, trustStorePassword);
+        CallHomeExecutor.execute(callhomeinfo);
+        String callHomeResponse = CallHomeExecutor.getMessage();
+        String formattedMessage = MessageFormatter.formatMessage(callHomeResponse, 180);
```

Comment:
I think we need some time between execute and print methods.

Reply:
some time mean?

Reply:
I mean after execute since it running in background, by the time we print the message, the process might not be finished yet